pkg/apis/autoscaling/v1/verticalpodautoscalercontroller_types.go

```diff
@@ -17,7 +18,38 @@ type VerticalPodAutoscalerSpec struct {
 	// +kubebuilder:validation:Minimum=0
 	PodMinMemoryMb     *float64 `json:"podMinMemoryMb,omitempty"`
 	RecommendationOnly *bool    `json:"recommendationOnly,omitempty"`
-	MinReplicas        *int64   `json:"minReplicas,omitempty"`
+	// +kubebuilder:validation:Minimum=1
+	MinReplicas *int64 `json:"minReplicas,omitempty"`
+	// +optional
+	ContainerOverrides ContainerOverrides `json:"containerOverrides,omitempty"`
+}
+
+// ContainerOverrides defines overrides for containers managed by the VerticalPodAutoscalerController
+type ContainerOverrides struct {
+
+	// Admission is the container overrides for the VPA's admission container
+	// +optional
+	Admission ContainerOverride `json:"admission,omitempty"`
+	// Recommender is the container overrides for the VPA's recommender container
+	// +optional
+	Recommender ContainerOverride `json:"recommender,omitempty"`
+	// Updater is the container overrides for the VPA's updater container
+	// +optional
+	Updater ContainerOverride `json:"updater,omitempty"`
+}
```

Comment:
I'm not sure I like what I called these, but now that I have it generating it's easy to change. 

Do we think this ends like it did over in CMA with being able to override the pod spec too? Do we like that interface e.g.  https://github.com/openshift/custom-metrics-autoscaler-operator/blob/ea0c16951aabdeaecba690d718d2304bc779bc5d/apis/keda/v1alpha1/kedacontroller_types.go#L98 ?